instagram/src/components/CommentSection/CommentInput.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function CommentInput(props) {
+    return (
+        <form onSubmit={props.addNewComment} onChange={props.handleChanges}>
+            <input
+              type = "text"
+              name = "comment"
```

Comment:
You have `newComment` on your state in CommentContainer, but you're calling `name="comment"` here in CommentInput. If you change name to `name="newComment"`, your handleChange function will work dynamically, rather than having to call this.setState 2x.